### DISCUSSION TITLE
Why can't mockoon save parameters?

### DISCUSSION THREAD

**Discussion by bewareilive** [mockoon/mockoon#676] [General]
Even if it's just as a note，
Recent time use has revealed that sometimes it is necessary to know what the query parameters are.
Usually just don't want to go to the code to see what parameters the interface has.
Just a note would be great，I don't want another postman.
Any future plans for this? or Is there a solution now?

**Comment by 255kb** [comment]
There is currently nothing on the roadmap for this. 
Mockoon only take cares of the response, not the request. It is not an API design tool.
You can use the "documentation" field to precise the query params needed.

**Reply by bewareilive** [reply]
> There is currently nothing on the roadmap for this. Mockoon only take cares of the response, not the request. It is not an API design tool. You can use the "documentation" field to precise the query params needed.
I agree with you.
That's how I use it now . But there is a problem, "documentation" field  is  single line,when there are many parameters or when using JSON objects.It is  inconvenient.
